tethys_gizmos/gizmo_options/cesium_map_view.py

```diff
@@ -461,8 +471,8 @@ class CesiumMapView(TethysGizmoOptions):
     # Set Cesium Default Release Version.
     cesium_version = "1.63.1"
 
-    def __init__(self, options={}, globe={}, view={}, layers=[], entities=[], terrain={}, models={}, clock={},
-                 height='100%', width='100%', draw=False, attributes={}, classes='', cesium_ion_token=''):
+    def __init__(self, options={}, globe={}, view={}, layers=[], entities=[], primitives=[], terrain={}, models={},
```

Comment:
There could potentially be some problems with the empty dicts/lists in the function definition. It is better practice to do:

```python
def __init__(self, options=None ...):
    self.options = options or {}
```

That way each instance gets its own default empty dict/list rather than having all instances share the same one.